android/shopify_buy_plugin/src/test/java/com/shopify/unity/buy/UnityMessageTest.java

```diff
@@ -0,0 +1,45 @@
+package com.shopify.unity.buy;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import org.junit.Test;
+
+import java.lang.reflect.Type;
+import java.util.Map;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+
+public class UnityMessageTest {
+
+    @Test
+    public void test_serialize() {
```

Comment:
Nit: Doesn't really matter but we can follow captial casing as a convention for naming test cases. Just going based off what we've done for the Mobile Buy SDK https://github.com/Shopify/mobile-buy-sdk-android/blob/master/MobileBuy/buy3-pay-support/src/androidTest/java/com/shopify/buy3/pay/PayCartTest.java